addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_63090","Costs of Value Loss of Debts from Customers",63090,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63091","Costs of Inventory Value Loss",63091,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63092","Costs of Intangible Assets Value Loss",63092,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63093","Costs of Value Loss of Long-Term Tangible Assets",63093,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63094","Costs of Value Loss of Other Assets",63094,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_6310","Provisions Costs",6310,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_6311","Costs of Fines",6311,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63120","Other Costs and Bank Taxes",63120,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63121","Representative Costs",63121,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63122","Daily Allowance",63122,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_63123","Support",63123,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5_general_administrative_expenses","False"
+"account_account_template_6400","Asset Disposition Losses",6400,"account.data_account_type_other_income","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_6_other_operating_results","False"
+"account_account_template_6401","Other Costs",6401,"account.data_account_type_other_income","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_6_other_operating_results","False"
```

Comment:
Accounts `6400` and `6401` seems to have incorrect type. It's now `Other Income`, yet their names (`... Losses`, `... Costs`) suggest that they are some sort of `Expenses`

```suggestion
"account_account_template_6401","Other Costs",6401,"account.data_account_type_expenses","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_6_other_operating_results","False"
```